### PR TITLE
Issue #662 - Multiple files upload

### PR DESCRIPTION
**Pull request by lbteles**

### CODE REVIEW

File: inc/form_answer.class.php [hunk 1]
```diff
@@ -643,19 +643,37 @@ public function saveAnswers($data) {
...
+                       'AND' => [
+                         'plugin_formcreator_forms_answers_id' => $formanswers_id,
+                         'plugin_formcreator_questions_id'     => $questionId
+                       ]
+                     ]);
+                     $answer->update([
+                        'id'     => $answer->getID(),
+                        'answer' => $answer_value,
+                     ], 0);
+                  }
```
Comment (btry) [inline inc/form_answer.class.php]: If $answer_value is not an array, I think we can convert it onto an array then handle it with the foreach.

Check your PR is writazble by the maitnainers of the plugin, and I'll be able to push this refactor myself.

Comment (lbteles) [reply inc/form_answer.class.php]: Yeah, is writazble.

**Comment by btry** [comment]
hummm.... I forgot about that : changing a commit closes a PR...

I changed the commit mesages of your 2 commits and added a new one with code refactor.. This is on your develop branch.

Open again a PR from your develop branch to run unit tests.

I also noticed a possible malfunction in the code. I'll test that next week. When you upload several files, only the last one will appear in the answer for the field, because in the foreach loop, we update the answer on each iteration. I think this update should be done after the foreach loop, and in a way which shows all uploads (maybe an implode() call).

**Comment by btry** [comment]
FYI I pushed these changes in my fork : https://github.com/pluginsGLPI/formcreator/pull/943

**Comment by alyssontkd** [comment]
Great job @lbteles !

TY!

**Comment by lbteles** [comment]
@alyssontkd, tamo junto mano velho!

**Pull request closed without merging**